src/views/Register.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react'
+import { withRouter, Link } from 'react-router-dom'
+import styled from 'styled-components'
+
+import mutate from '../utils/mutate'
+import graphql from 'babel-plugin-relay/macro'
+
+import { Form, Container } from 'react-bootstrap'
+import { Wrapper } from '../styles/Wrapper'
+import { Button } from '../styles/Button'
+import { Title } from '../styles/Title'
+import { ErrorMessage } from '../styles/Error'
+
+const mutation = graphql`
+  mutation RegisterMutation($input: UserCreateInput!){
+    userCreate(input: $input){
+            user {
+                id
+            }
+        }
+    }
+`
+
+const Register = ({ history, props }) => {
+  const [data, setData] = useState([])
+  const { username, email, password } = data
+  const [error, setError] = useState('none')
+
+  const validateForm = () => {
+    return data.email && data.password && data.username
+  }
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+
+    const variables = {
+      input: {
+        username: username,
+        email: email,
+        password: password
+      }
+    }
+
+    mutate(mutation, variables)
+      .then(() => { history.push('/login') })
+      .catch((e) => {
+        const message = e[0].message.substring(96)
```

Comment:
We agreed not to do this! Please include the full message, and use a css ellipsis to truncate. This is not scalable and should be corrected on the backend, not massaged on the front-end.